types/mssql/index.d.ts

```diff
@@ -208,6 +208,13 @@ export interface config {
     beforeConnect?: (conn: tds.Connection) => void
 }
 
+export declare class MSSQLError extends Error {
+    constructor(message: Error | string, code?: string);
+    public code: string;
+    public name: string;
```

Comment:
`code` and `name` are inherited from `Error`:
https://nodejs.org/docs/latest-v14.x/api/errors.html#errors_error_code

Indeed, they are not defined in https://github.com/tediousjs/node-mssql/blob/master/lib/error/mssql-error.js

On the other hand, they do not seem to be included in the the defintitions for `node` as they should be.
The correct way to solve this would be to add them to the definitions for `node`, but I guess that is outside the scope of this PR.

Reply:
Actually the `node` docs say `Error` defines `code` and `message`, but not `name`. But then typescript says `Error` has `name` instead of `code`. So I'll have to leave both for now :/